tests/suites/cloud/tests/preload/index.js

```diff
@@ -9,21 +9,18 @@ module.exports = {
     // if test is being done on physical DUT via the testbot, check that the preloaded application is working
     if(this.workerContract.workerType !== `qemu`){
       // we should be able to see the app starting.
-      const ip = await this.worker.ip(this.link);
-      
-      // create tunnel to DUT port 80
-      console.log(`Creating tunnel to DUT port 80...`)
-      await this.worker.createTunneltoDUT(this.link, 80, 8899);
-      await this.utils.waitUntil(async () => {
-        console.log(`Checking preloaded app is running... `)
-        let page = await request({
-          method: 'GET',
-          uri: `http://${ip}:80`,
-        })
-        return page.includes("Welcome to balena!"); 
-      }, false);
+  
+      await this.utils.waitUntil(
+        async () => {
+          console.log(`Checking preloaded app has started`)
+          let result = await this.worker.executeCommandInHostOS(
+            'journalctl -a | grep ": HELLO_WORLD"',
```

Comment:
Good one dropping the hello-world where it wouldn't matter but does this take us away from testing actual apps which is something we want to actually do?

I know it doesn't matter in this test, but atleast the app was getting tested in this process.

Reply:
Yeah it is a shame in some respects - this test using bellow world actually did lead to a hello world bug fix at one point actually.

But im not sure it has a place here in this test.

Reply:
Ahh true that. Good thing we have an improvement about this that alexgg posted about. So we know it's being tracked for later.